exploration/validate_outputs.md

```diff
@@ -0,0 +1,263 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.3
+  kernelspec:
+    display_name: venv
+    language: python
+    name: python3
+---
+
+## Validate output stats
+
+In this notebook we're taking a look at our output raster stats and comparing against the outputs from other raster stats calculation functions. We would expect our outputs to be the same as the `zonal_stats` outputs, and see some moderate differences with the `exactextract` outputs.
+
+```python
+%load_ext jupyter_black
+%load_ext autoreload
+%autoreload 2
+```
+
+```python
+from exactextract import exact_extract
+from rasterstats import zonal_stats
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, text
+import os
+import pandas as pd
+import geopandas as gpd
+import rioxarray as rxr
+import numpy as np
+
+# os.chdir("..")
+from src.utils.cloud_utils import get_cog_url
+from src.utils.raster_utils import upsample_raster, prep_raster
+
+load_dotenv()
+```
+
+```python
+AZURE_DB_PW_PROD = os.getenv("AZURE_DB_PW_PROD")
+MODE = "prod"
+
+adm_level = 2
+iso3 = "AFG"
+date = "2024-01-01"
+stats = ["mean", "median", "min", "max"]
+engine = create_engine(
+    f"postgresql+psycopg2://chdadmin:{AZURE_DB_PW_PROD}@chd-rasterstats-prod.postgres.database.azure.com/postgres"
+)
+gdf = gpd.read_file(f"data/{iso3.lower()}/{iso3.lower()}_adm{adm_level}.shp")
```

Comment:
again -- need this data folder

Reply:
Using the `os.chdir("..")` sets the working directory one level higher, so that this path works.